src/assets/images/exchange/index.js

```diff
@@ -16,9 +17,10 @@ import faastLogo from './settingsExchangeFaast.png'
 import foxExchangeLogo from './settingsExchangeFoxExchange.png'
 import godexLogo from './settingsExchangeGodex.png'
 import shapeshiftLogo from './settingsExchangeShapeshift.png'
+import switchainLogo from './settingsExchangeSwitchain.png'
 import totleLogo from './settingsExchangeTotle.png'
 
-export function getSwapPluginIcon (pluginId: string) {
+export function getSwapPluginIcon(pluginId: string) {
```

Comment:
Can you run `yarn format`? This looks like a formatting glitch.

Reply:
Done